scripts/check_frontend_coverage.py

```diff
@@ -321,6 +321,7 @@
     'question-creation.service.ts',
     'question-difficulty-selector.directive.ts',
     'question-editor.directive.ts',
+    'question-misconception-editor.directive.ts',
```

Comment:
You should create unit tests for this file, as we've changed some things related to frontend coverage

Reply:
Removed this.